ci/github-script/merge.js

```diff
@@ -86,6 +88,10 @@ async function handleMergeComment({ github, body, node_id, reaction }) {
   )
 }
 
+// Caching the list of team members saves API requests when running the bot on the schedule and
```

Comment:
Is it more accurate to say we're [memoizing](https://en.wikipedia.org/wiki/Memoization) the members, or fetching them lazily? We aren't writing to  a persistent cache.


```suggestion
// Memoizing team members saves API requests when running the bot on the schedule and
```